src/utils/fields.js

```diff
@@ -126,6 +126,25 @@ const PREVIEW_FIELDS_TO_FETCH = [
     FIELD_IS_DOWNLOAD_AVAILABLE,
 ];
 
+// Fields needed to get versions for a file in activity feed
+const FEED_FILE_VERSIONS_FIELDS_TO_FETCH = [
+    FIELD_AUTHENTICATED_DOWNLOAD_URL,
```

Comment:
@karelee7, can we add a very brief comment next to the particularly expensive fields?